executor/analyze.go

```diff
@@ -273,6 +268,54 @@ func (e *AnalyzeExec) handleResultsError(ctx context.Context, concurrency int, n
 	return err
 }
 
+func (e *AnalyzeExec) handleResultsErrorWithConcurrency(ctx context.Context, statsConcurrency int, needGlobalStats bool,
+	subSctxs []sessionctx.Context,
+	globalStatsMap globalStatsMap, resultsCh <-chan *statistics.AnalyzeResults) error {
+	partitionStatsConcurrency := len(subSctxs)
```

Comment:
What's relation between `statsConcurrency` and `partitionStatsConcurrency`?

Reply:
`statsConcurrency` is the concurrency for building `analyze results` while `partitionStatsConcurrency` is the concurrency to save results into storage.